### PR TITLE
Feat/header

### PR DESCRIPTION
**Pull request by mateosierra17**

### CODE REVIEW

File: components/Header/header.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.responsive-menu {
```
Comment (calixfar) [inline components/Header/header.css:1]: Please indent the whole content to 2 spaces.

File: app/layout.tsx [hunk 2]
```diff
@@ -7,8 +7,8 @@ import '../styles/globals.css'
 
 
 export const metadata: Metadata = {
-  title: "Luis Santiago's Blog",
-  description: 'dedicated blog to talk about topics related to: tecnology, software development, frontend, backend, clean code, data structure, design patterns and also other topics more close to the life :)',
```
Comment (calixfar) [inline app/layout.tsx:11]: Why did you remove keywords on the description? This is content is aimed to better seo

File: components/Header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+export default function Header() {
+  return (
+    <header className="max-w-screen border-b border-solid border-black p-8 flex justify-between content-center">
+      <nav className="header-links">
+        <Link className="font-bold p1" href="./">
+          About
+        </Link>
+      </nav>
+
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove empty line

File: components/Header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <h1 className="text-2xl font-bold hover:text-yellow-600">
+        <Link href="/">Synergy Wire Blog </Link>
+      </h1>
+
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove empty line

File: components/Header/index.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        <Link
+          className="text-slate-400 organization-link"
+          href="https://github.com/SynergyWire"
+          target="_blank"
+        >
+          <Icon name={ICON_KEYS.GITHUB} />
+        </Link>
+      </div>
+      <label className="responsive-menu" htmlFor="toggle-button">
+
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove empty line

File: components/Header/index.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+        <input type="checkbox" id="toggle-button" />
+
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove empty line

File: components/Header/index.tsx [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+        <div className="responsive-menu__icon-switch">
+
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove empty line

File: components/Header/index.tsx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+          <svg className="menu-icon-burger"
```
Comment (calixfar) [inline components/Header/index.tsx]: Could you create a component Icon to these svg. Follow the example on the components folder.

File: components/Header/index.tsx [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+          </li>
+          <li>
+            <Link className="font-bold p1 hover:text-yellow-600" href="./">
+              Authors
+            </Link>
+          </li>
+        </ul>
+      </label>
+    </header>
+  );
```
Comment (calixfar) [inline components/Header/index.tsx]: Remove ;

File: components/Header/header.css [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+    display: none;
+}
+
```
Comment (calixfar) [inline components/Header/header.css:4]: Remove empty line. There must just be one empty line between blocks.

**Review by calixfar — CHANGES_REQUESTED** [review]

File: app/layout.tsx (hunk 2 above)
Comment (mateosierra17) [reply app/layout.tsx:11]: The content will be discussed and created in the next meeting, in order to get a clearer concept of what synergy wire will be

File: utils/types.ts [hunk 11]
```diff
@@ -1 +1 @@
-export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER'
\ No newline at end of file
+export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER' |'MENU'|'CLOSE'
```
Comment (calixfar) [inline utils/types.ts]:
```suggestion
export type IconKey = 'GITHUB' | 'LINKEDIN' | 'TWITTER' | 'MENU' | 'CLOSE'
```
Add blank spaces between options

File: components/icons/Icon.tsx [hunk 12]
```diff
@@ -1,13 +1,17 @@
...
+import IconX from './Icon-x'
 
 export default function Icon({ name }: { name: IconKey }) {
   const ICONS = {
     GITHUB: IconGithub,
     LINKEDIN: IconLinkedin,
-    TWITTER: IconTwitter
+    TWITTER: IconTwitter,
+    MENU: IconMenu,
+    CLOSE: IconX
```
Comment (calixfar) [inline components/icons/Icon.tsx]: Is it not better if you named the icon as IconClose?

File: components/Header/index.tsx [hunk 13]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
```
Comment (calixfar) [inline components/Header/index.tsx]: Please keep a format, you are calling the imports with double quotes but on other files it is just single quote. I would rather have one quote and no ;

**Review by calixfar — APPROVED** [review]

**Pull request merged**